.gitignore

```diff
@@ -15,3 +15,4 @@ target/
 tmp/
 # Ignore .pb files generated by profiling
 *.pb
+tools/smoke-test/Cargo.lock
```

Comment:
Is the idea of gitignorning this lock file that we want to see this test fail if dependencies update in a way that breaks us?